app.js

```diff
@@ -0,0 +1,75 @@
+const hideList = document.querySelector(`#hideList`);
+const description = document.querySelector('#description');
+const descriptionInput = document.querySelector('#descriptionInput');
+const descriptionButton = document.querySelector('#descriptionButton');
+const addItemInput = document.querySelector('#addItemInput');
+const addItemButton = document.querySelector('#addItemButton');
+const myList = document.querySelector(`#myList`);
+
+// kod z cwiczenia
+//zmienilem event.target.tagName == 'BUTTON' na event.target.className === 'removeButton'
+// dodalem class 'removeButton' do elementów button w HTML wewnatrz li
+
+const listElements = document.getElementsByTagName('li');
+const listDiv = document.querySelector(`.list`);
+
+
+
+
+listDiv.addEventListener('click', (event) =>{
+	if(event.target.className === 'removeButton') {
+		let li = event.target.parentNode;
+		let ul = li.parentNode;
+		ul.removeChild(li);
+	}
+});
+
+// koniec kodu z cwiczenia
+
+
+// zmiana opisu listy
+descriptionButton.addEventListener('click', () => {
+	description.textContent = `${descriptionInput.value} :`;
+	descriptionInput.value = '';
+});
+
+//ukrywanie/odsłanianie listy
+
+hideList.addEventListener('click', () => {
+	if (listDiv.style.display === 'block' || listDiv.style.display === ''  ) {
```

Comment:
Sporo niepotrzebnej logiki. Lepszym rozwiązanie byłoby togglowanie określonej klasy np. `listDiv.classList.toggle('is-visible`)`, dzieki temu nie musiałbyś sprawdzać czy element jest widoczny, czy nie, po prostu klasa byłaby automatycznie dodawana lub usuwana.

Wtedy mógłbyś rozwiązać kwestię `textContent` w ten sposób:

`hideList.textContent = listDiv.classList.contains('is-visible') ? 'Hide list' : 'Show list';`

To jest tzw. ternary conditional operator. Mogę Ci wyjaśnić dokładnie jak działa przez tel. ale pewnie sam złapiesz ;)

Reply:
Generalnie zamiast ustawiać `element.style =` lepszą praktyką jest dodawanie / usuwanie / toggle'owanie `CSSowej` klasy, która dodaje odpowiednie style